app/views/site/contribute.erb

```diff
@@ -0,0 +1,130 @@
+
+      <ol>
+        <li><strong>The website</strong>: Its here that we can improve exercism features or layout.</li>
+        <li><strong>Command line interface</strong>: Fetch exercises and submit solutions.</li>
+        <li><strong>Language track repositories</strong>: Each language has its own repository with available problems.</li>
+        <li><strong>Problem metadata</strong>: Contains problem descriptions.</li>
+        <li><strong>Problems API</strong>: Delivers exercises, consumed by the Exercism command-line client, as well as the Exercism website.</li>
+        <li><strong>Rikki- the robot</strong>: Provides automated feedback on certain exercises.</li>
+      </ol>
+    </p>
+  </section>
+
+  <hr>
+
+  <section class="contributing-areas">
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Make Up Exercises</h2>
+
+          <p>
+            If you have an idea for an exercise you can go to the <a href="https://github.com/exercism/x-common">x-common</a> repository, there you will find existing exercises and can check if your exercise is not already defined.
+          </p>
+
+           <p>
+            You can check <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#implementing-a-completely-new-exercise">here</a> to learn how to implement a new exercise.
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Translate Exercises</h2>
+
+          <p>
+            Translating an exercise to your favorite language is a great way to help the comunity, track maintainers will be more than happy to give all the support needed so you can complete your first contribution.
+          </p>
+
+          <p>
+            A detailed guide can be seen <a href="https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md#porting-an-exercise-to-another-language-track">here</a> if you want to give it a try!
+          </p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Review Pull Requests</h2>
+
+          <p>
+            Reviewing Pull Requests is a great way to both learn and teach a programming language, the track maintainers will really appreciate your help!
+          </p>
+
+          <p>
+             You can search your language repository <a href="https://github.com/exercism/">here</a>.
+          </p>
+        </div>
+
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Canonical Data</h2>
+
+          <p>What is this section about?</p>
+          <p>A link to further instructions.</p>
+        </div>
+      </div>
+
+      <div class="col-md-4">
+        <div class="contributing-section">
+          <h2>Help Triage Issues</h2>
+
+          <p>What is this section about?</p>
```

Comment:
_We've got a huge number of old issues that aren't particularly actionable, and new issues come in all the time._

_Is an issue no longer relevant? Can it be closed? Do we need more information? If it's a bug, is it reproducible? If it's a discussion and we need to make some sort of decision, then it should be labeled as such._

Maybe we can link to this discussion for now: https://github.com/exercism/discussions/issues/20